geode-core/src/main/java/org/apache/geode/internal/tcp/Connection.java

```diff
@@ -1763,9 +1763,13 @@ private static boolean isIgnorableIOException(Exception e) {
     }
 
     msg = msg.toLowerCase();
-    return msg.contains("forcibly closed")
-        || msg.contains("reset by peer")
-        || msg.contains("connection reset");
+
+    if (e instanceof SSLException && msg.contains("status = closed")) {
```

Comment:
The commit message references reverting the direct buffer changes. Is this change intentional because it doesn't look like part of the reverted commit?

Reply:
I'm confused by this comment because I very clearly included changing back to non-direct buffers in Connection and NioSslEngine in this commit.